src/components/theme/View/View.jsx

```diff
@@ -227,8 +228,55 @@ class View extends Component {
           {this.props.content.language && (
             <html lang={this.props.content.language.token} />
           )}
-          <title>{this.props.content.title}</title>
-          <meta name="description" content={this.props.content.description} />
+          <title>
+            {this.props.content.seo_title || this.props.content.title}
+          </title>
+          <meta
+            name="description"
+            content={
+              this.props.content.seo_description ||
+              this.props.content.description
+            }
+          />
+
+          <meta
+            property="og:title"
+            content={
+              this.props.content.opengraph_title ||
+              this.props.content.seo_title ||
+              this.props.content.title
+            }
+          />
+          <meta
+            property="og:url"
+            content={
+              this.props.content.seo_canonical_url ||
+              toPublicURL(this.props.content['@id'])
+            }
+          />
+          {this.props.content.image?.scales?.large?.download ||
+            (this.props.content.opengraph_image?.scales?.large?.download && (
+              <meta
+                property="og:image"
+                content={toPublicURL(
+                  this.props.content.opengraph_image?.scales?.large?.download ||
+                    this.props.content.image?.scales?.large?.download,
+                )}
+              />
+            ))}
```

Comment:
I'd suggest to implement image sizes in opengraph
ref: https://developers.facebook.com/docs/sharing/best-practices/?locale=en_US#images

```suggestion
          {this.props.content.image?.scales?.large?.download ||
            (this.props.content.opengraph_image?.scales?.large?.download && (
              <>
                <meta
                  property="og:image"
                  content={toPublicURL(
                    this.props.content.opengraph_image?.scales?.large?.download ||
                      this.props.content.image?.scales?.large?.download,
                  )}
                />
                <meta
                  property="og:image:width"
                  content={
                    this.props.content.opengraph_image?.scales?.large?.width ||
                        this.props.content.image?.scales?.large?.width
                  }
                />
                <meta
                  property="og:image:height"
                  content={
                    this.props.content.opengraph_image?.scales?.large?.height ||
                        this.props.content.image?.scales?.large?.height
                  }
                />
              </>
            ))}
```